### PR TITLE
Refine polish section impact, define milestones

### PR DESCRIPTION
**Pull request by eholk**
This change elaborates the desired impact from the polish area and also
sets an initial set of milestones to bring about this impact. We will
continue to refine these over time.

### CODE REVIEW

File: src/vision/roadmap/polish.md [hunk 1]
```diff
@@ -2,7 +2,37 @@
...
+
+| Milestone                                  | State | Key participants |
+| ---                                        | ---   | ---              |
+| Precise generator captures                 | 🦀    | [eholk]          |
+| ↳ Prototype                                | 🦀    | [eholk]          |
+| ↳ Documented in Rust Reference             | 🦀    | [eholk]          |
+| ↳ [Lang team] signoff                      | 💤    | [Lang team]      |
+| ↳ Stabilized                               | 💤    | [eholk]          |
+| Lint: [Large copies], large generators     | 💤    | |
+| ↳ [Lang team] initiative proposal          | 💤    | |
```
Comment (tmandry) [inline src/vision/roadmap/polish.md:26]: Precise generator captures should have this too, I think

File: src/vision/roadmap/polish.md [hunk 2]
```diff
@@ -2,7 +2,37 @@
...
+| ↳ Implementation                           | 💤    | |
+| Lint: [Must not suspend]                   | 💤    | |
+| ↳ Landed the [RFC]                         | 💤    | |
```
Comment (tmandry) [inline src/vision/roadmap/polish.md:29]: Needs implementation?

File: src/vision/roadmap/polish.md [hunk 3]
```diff
@@ -2,7 +2,37 @@
...
+| Lint: [Blocking in async context]          | 💤    | |
```
Comment (tmandry) [inline src/vision/roadmap/polish.md:30]: Needs RFC and implementation

**Review by tmandry — APPROVED** [review]

File: src/vision/roadmap/polish.md (hunk 2 above)
Comment (eholk) [reply src/vision/roadmap/polish.md:29]: I'll rephrase as "Implemented the RFC." The RFC is already merged, so I think all we need to do is implement what the RFC says.

File: src/vision/roadmap/polish.md (hunk 1 above)
Comment (eholk) [reply src/vision/roadmap/polish.md:26]: Done.

File: src/vision/roadmap/polish.md (hunk 3 above)
Comment (eholk) [reply src/vision/roadmap/polish.md:30]: Done

**Review by nikomatsakis — APPROVED** [review]
Looks good to me

**Pull request merged**